checker/run

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
+
+set -uex
+
+. "$(dirname "$0")"/../options
+
+if [ -z "$pattern_extractor" ] | [ -z "$message" ]; then
```

Comment:
`[ -z "$pattern_extractor" -o -z "$message" ]`

Note the single | in the middle, which means something else entirely.